src/components/MarketList/index.js

```diff
@@ -93,13 +91,11 @@ class MarketList extends Component {
         <div className="market__header">
           <h2 className="market__title">{market.eventDescription.title}</h2>
           {isOwner &&
-          !isResolved && (
-            <div className="market__control">
-              <a href={`/#${resolveUrl}`} onClick={e => this.handleViewMarketResolve(e, resolveUrl)}>
-                Resolve
-              </a>
-            </div>
-          )}
+            !isResolved && (
+              <div className="market__control">
+                <a onClick={e => this.handleViewMarketResolve(e, resolveUrl)}>Resolve</a>
```

Comment:
Don't we need a href for `<a>` to work semantically?

Reply:
@andre-meyer hmm, there is an eslint warning about this:
`Visible, non-interactive elements should not have mouse or keyboard event listeners (jsx-a11y/no-static-element-interactions)`
Probably we need to change it

Reply:
how about the good old `href="javascript:void(0)"`